lib/CompilerPluginSupport/CMakeLists.txt

```diff
@@ -56,4 +77,7 @@ if(SWIFT_SWIFT_PARSER)
       DESTINATION "lib${LLVM_LIBDIR_SUFFIX}"
       COMPONENT compiler)
 
+    swift_install_in_component(DIRECTORY "${module_base}"
```

Comment:
Do we still need line 66-72?

Reply:
I think that's installing the dylib itself

Reply:
They're both installed into `lib`, without the platform, because they're host content.

Reply:
Sorry I meant to say `lib/swift`. Currently the library is installed to `lib` but the module is installed to "lib${LLVM_LIBDIR_SUFFIX}/swift".

Reply:
Both are going into `lib` with this pull request. We could decide that they go somewhere else, but there's some precedent for using `lib`: `lib_InternalSwiftScan.dylib`, `lib_InternalSwiftStaticMirror.dylib`, `libsourcekitdInProc.dylib` and `libswiftDemangle.dylib`, all host libraries, all go there.

Reply:
`lib` makes sense to me.